### DISCUSSION TITLE
Installation of vercel@25.2.3 is crashing the Github Actions

### DISCUSSION THREAD

**Discussion by guscsales** [vercel/vercel#8062] [Help]
Hey guys!

Currently, I'm using the vercel as a global package inside my CI to deploy to Vercel's platform.

```
- name: '[Development] Deploy Affected Apps'
        run: |
          yarn global add vercel
          yarn deploy:all
```

After the new version (i.e.: `25.2.3`) I notice the GA start crashing:
![image](https://user-images.githubusercontent.com/5130296/176768467-b0740431-3c31-4750-9d5a-9457507acd3a.png)

Then, testing it locally I notice on this new minor now we need to choose the version of two packages, a.k.a `@vercel/next` and `@vercel/redwood`:
![image](https://user-images.githubusercontent.com/5130296/176768286-79bd771d-7b60-4c6b-9b8a-67aea9e23e3b.png)

![image](https://user-images.githubusercontent.com/5130296/176768304-163e72a7-9d83-4307-ad4b-224243d95b01.png)

Since my global installation is running inside the GA, I can't choose an option by using terminal interactions. So my question: is possible to add those packages with some version set by default? 

Thanks a lot!

**Comment by amyegan** [comment]
Hi @guscsales. There was an [npm incident](https://status.npmjs.org/incidents/6wr25yb0b2dd) yesterday that caused some issues with `@vercel/next@3.1.2` and other packages, but it should be resolved now. Are you still having this problem?
